modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/CheckboxMultiple/CheckboxMultiple.es.js

```diff
@@ -237,4 +237,89 @@ describe('Field Checkbox Multiple', () => {
 
 		expect(handleFieldEdited).toHaveBeenCalled();
 	});
+
+	it('checks the value if there is a value', () => {
+		render(
+			<CheckboxMultipleWithProvider
+				options={[
+					{
+						label: 'Option 1',
+						value: 'option1',
+					},
+					{
+						label: 'Option 2',
+						value: 'option2',
+					},
+					{
+						label: 'Option 3',
+						value: 'option3',
+					},
+				]}
+				predefinedValue={['option1', 'option2']}
+				spritemap={spritemap}
+				value={['option3']}
+			/>
+		);
+
+		expect(document.querySelector('[value="option1"]').checked).toBe(false);
+		expect(document.querySelector('[value="option2"]').checked).toBe(false);
+		expect(document.querySelector('[value="option3"]').checked).toBe(true);
+	});
+
+	it('checks the predefinedValue if there is no value', () => {
+		render(
+			<CheckboxMultipleWithProvider
+				options={[
+					{
+						label: 'Option 1',
+						value: 'option1',
+					},
+					{
+						label: 'Option 2',
+						value: 'option2',
+					},
+					{
+						label: 'Option 3',
+						value: 'option3',
+					},
+				]}
+				predefinedValue={['option1', 'option2']}
+				spritemap={spritemap}
+				value={[]}
+			/>
+		);
+
+		expect(document.querySelector('[value="option1"]').checked).toBe(true);
+		expect(document.querySelector('[value="option2"]').checked).toBe(true);
+		expect(document.querySelector('[value="option3"]').checked).toBe(false);
```

Comment:
- Prefer use [testing library queries](https://testing-library.com/docs/queries/about/), because they enforce the code accessibility 
- Prefer assertions methods from [jest-dom](https://github.com/testing-library/jest-dom), because they provide a more accurate error message in case of failure
```suggestion
		expect(getByLabelText('Option 1')).toBeChecked();
		expect(getByLabelText('Option 2')).toBeChecked();
		expect(getByLabelText('Option 3')).not.toBeChecked();
```
**Note:**
- maybe the `getByLabelText()` is not the proper query to use, in that case try `getByText()`
- depends on the component's implementation you won't be able to use testing library queries